examples/installation_on_windows/README.md

```diff
@@ -0,0 +1,57 @@
+# Install SLLM on Windows Machines
+
+## 1 Windows Subsystem for Linux
+To install the SLLM dependencies, Ubuntu 20.04 is required by the installation guidance states. Packages, like vllm, cannot be installed on Windows. Therefore, the Windows machines require a sub-system to run Linux. Here, it's recommended downloading the Ubuntu app from Microsoft store and the WSL connection will allow to you install everything needed for. If you have more familiar to start Linux environment, please feel free to go ahead and share your experience with us. Generally speaking, it is a developed and stable app except WSL connection doesn't support GUI, and sometimes it can be nasty to set variable path.
+
+![alt text](Ubuntu-app.png)
+
+Description:
+Install a complete Ubuntu terminal environment in minutes with Windows Subsystem for Linux (WSL). Develop cross-platform applications, improve your data science or web development workflows and manage IT infrastructure without leaving Windows.
+
+Key features:
+  - Efficient command line utilities including bash, ssh, git, apt, npm, pip and many more
+  - Manage Docker containers with improved performance and startup times
+  - Leverage GPU acceleration for AI/ML workloads with NVIDIA CUDA
+  - A consistent development to deployment workflow when using Ubuntu in the cloud
+  - 5 years of security patching with Ubuntu Long Term Support (LTS) releases
+
+Ubuntu currently provides the 24.04.1 LTS release. When new LTS versions are released, Ubuntu can be upgraded once the first point release is available. This can be done from the command line by using:
```

Comment:
Hi, could you please avoid specifying the current Ubuntu release version? Since we cannot predict the Ubuntu release version users will have when using ServerlessLLM, it would be better to simply provide instructions on how to upgrade.

Reply:
Thank you for this advice. I removed the detailed descriptions (version number) and left only the key information to reduce ambiguity.